doc/nrf/protocols/matter/end_product/ecosystems_certification.rst

```diff
@@ -0,0 +1,61 @@
+.. _ug_matter_ecosystems_certification:
+
+Ecosystems certification
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Matter stack provided in the |NCS| works with commercial ecosystems that are compatible with official Matter implementation.
+The Matter product is called compatible with Matter implementation once it passed relevant Matter certification.
+
+However some of the ecosystem providers require passing additional certification to verify that Matter product is able to work with their applications and support their features.
```

Comment:
I think the word "require" here and in line 21 is a bit too strict (e.g. Google cert is not mandatory afaik). I think we can say that ecosystems offer additional certification programs to ensure that the product works well with the given ecosystem or correctly implement additional proprietary to matter features.

Reply:
Fair point, applied.